db/migrations/20230127191659-add-contract-params-to-network-table.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    const integerParam = {
+      type: Sequelize.BIGINT,
+      allowNull: true
```

Comment:
If I'm not mistaken, `allowNull` is true by default, so we don't need to pass it